hybridse/src/passes/physical/long_window_optimized.cc

```diff
@@ -231,7 +231,9 @@ bool LongWindowOptimized::OptimizeWithPreAggr(vm::PhysicalAggrerationNode* in, i
         LOG(ERROR) << "Fail to create PhysicalReduceAggregationNode: " << status;
         return false;
     }
+    LOG(INFO) << "[LongWindowOptimized] Before transform sql:\n" << (*output)->GetTreeString();
```

Comment:
DEBUG log ?

Reply:
Forgot to discuss this in the PR review. 

For testing and verification, we need some log information to see whether we are really using `long_window` optimization or not. 

Since the `log` will only print during `deploy`, it is kind of acceptable. What do you think?

If it is too annoying, can change to `Debug` log.

Reply:
if only in deploy, it's no matter